Source/JavaScriptCore/wasm/WasmLLIntPlan.cpp

```diff
@@ -131,11 +131,15 @@ void LLIntPlan::didCompleteCompilation()
             if (m_moduleInformation->usesSIMD(i))
                 JIT_COMMENT(jit, "SIMD function entrypoint");
             JIT_COMMENT(jit, "Entrypoint for function[", i, "]");
-            CCallHelpers::Address calleeSlot(CCallHelpers::stackPointerRegister, CallFrameSlot::callee * static_cast<int>(sizeof(Register)) - prologueStackPointerDelta());
-            jit.storePtr(CCallHelpers::TrustedImmPtr(CalleeBits::boxNativeCallee(m_calleesVector[i].ptr())), calleeSlot.withOffset(PayloadOffset));
-#if USE(JSVALUE32_64)
-            jit.store32(CCallHelpers::TrustedImm32(JSValue::NativeCalleeTag), calleeSlot.withOffset(TagOffset));
-#endif
+            {
+                // Debug assertion
```

Comment:
Is this part still WIP? (Or do we need the non-`good` path to exist?)

Reply:
This just adds a debug assertion to check that the callee was set correctly. Eventually we can just remove the whole thunk, once this patch has had an opportunity to get fuzzed